recipes/z5py/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.0.0" %}
+{% set checksum = "864209f54934bc8de064c4be403f8e49d532574334ae6bce76d193659866a87d" %}
+
+package:
+  name: z5py
+  version: {{ version }}
+
+source:
+    url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - boost-cpp
+    - numpy
+    - xtensor >=0.17.1,<0.18
+    - xtensor-python >=0.20,<0.21
```

Comment:
Are these things we should be pinning globally or do these just work well with the current state of z5py?

Reply:
I think it would make sense to pin these globally.
@SylvainCorlay recommended to pin to these versions because 
`xtensor` and `xtensor-python` API is still a bit unstable and can change with the next minor release.

Reply:
- xtensor packages follow semver for the API. >=0.17.1,<0.18 is a means to specify ^0.17.1.
- This means that z5 may not work with e.g. xtensor 0.18.